### PR TITLE
Refine/add `lightbulb` icons

### PR DESCRIPTION
**Pull request by danielbayley**

### CODE REVIEW

File: icons/lightbulb-2.svg [hunk 1]
```diff
@@ -0,0 +1,14 @@
...
+  height="24"
+  viewBox="0 0 24 24"
+  fill="none"
+  stroke="currentColor"
+  stroke-width="2"
+  stroke-linecap="round"
+  stroke-linejoin="round"
+>
+  <line x1="9" x2="15" y1="16" y2="16" />
+  <path d="M9 20c0 1.1.9 2 2 2h2c1.1 0 2-.9 2-2v-6c0-1.2.7-1.7 1.5-2.5 1-.9 1.5-2.2 1.5-3.5 0-3.3-2.7-6-6-6S6 4.7 6 8c0 1 .2 2.2 1.5 3.5.7.7 1.5 1.3 1.5 2.5v6z" />
```
Comment (jguddas) [inline icons/lightbulb-2.svg:13]:
```suggestion
  <path d="M9 16h6"/>
  <path d="M9 20a2 2 0 0 0 2 2h2a2 2 0 0 0 2-2v-6c0-1.2.7-1.7 1.5-2.5a5.8 5.8 0 1 0-9 0c.7.7 1.5 1.3 1.5 2.5v6"/>
```

File: icons/lightbulb.svg [hunk 2]
```diff
@@ -11,5 +11,5 @@
 >
   <line x1="9" x2="15" y1="18" y2="18" />
   <line x1="10" x2="14" y1="22" y2="22" />
-  <path d="M15.09 14c.18-.98.65-1.74 1.41-2.5A4.65 4.65 0 0 0 18 8 6 6 0 0 0 6 8c0 1 .23 2.23 1.5 3.5A4.61 4.61 0 0 1 8.91 14" />
+  <path d="M15 14c0-1.2.7-1.7 1.5-2.5 1-.9 1.5-2.2 1.5-3.5 0-3.3-2.7-6-6-6S6 4.7 6 8c0 1 .2 2.2 1.5 3.5.7.7 1.5 1.3 1.5 2.5" />
```
Comment (jguddas) [inline icons/lightbulb.svg:14]:
```suggestion
  <path d="M10 22h4"/>
  <path d="M9 18h6"/>
  <path d="M15 14a3.34 3.34 0 0 1 1.5-2.5 5.8 5.8 0 1 0-9 0A3.5 3.5 0 0 1 9 14"/>
```

Comment (danielbayley) [reply icons/lightbulb.svg:14]: Hmm, these changes result in some less than pretty curvature, and non pixel prefect dimensions of 11.6002x11.9594px…

Untitled

Untitled

Comment (jguddas) [reply icons/lightbulb.svg]: Oh, you are right.

This should be better:
```suggestion
  <path d="M10 22h4" />
  <path d="M9 18h6" />
  <path d="M15 14c0-1.2.7-1.7 1.5-2.5a5.8 5.8 0 1 0-9 0c.7.7 1.5 1.3 1.5 2.5" />
```

Comment (danielbayley) [reply icons/lightbulb.svg]: Does it matter that this makes the bulb 11.6002x11.9594px, as opposed to 12x12px…?

File: icons/lightbulb-2.svg
Comment (karsa-mistmere) [inline icons/lightbulb-2.svg:1]: Not really a fan of this one, it just doesn't quite look like a lightbulb to me with visible screw. 🙈

**Comment by danielbayley** [comment]
Hmm, what do we think of these alternatives/variations…? @karsa-mistmere @jguddas @ericfennis

Untitled

Untitled

**Comment by ericfennis** [comment]
I agree with @karsa-mistmere, I think this one does not fit very well with the other icons.
I also doubt the other variations you provided. I think it's better to move this back to an issue/discussion so we can discuss or iterate on this idea more.

**Comment by danielbayley** [comment]
> I think it's better to move this back to an issue/discussion so we can discuss or iterate on this idea more.

Ok, moved to #1196… But the original point of this PR was also to fix the existing `lightbulb` icons… As they are slightly off grid… @ericfennis See latest `push`…

**Pull request closed without merging**